### PR TITLE
让card的action更容易点击

### PR DESCRIPTION
**Pull request by lhx6538665**
不改的话太难点击了，很容易点不上

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by lhx6538665** [comment]
望批准

**Comment by afc163** [comment]
样式也要改，只改标签会导致点不到。

**Comment by lhx6538665** [comment]
我又修改了一下

File: components/card/style/index.less [hunk 1]
```diff
@@ -155,7 +155,8 @@
           line-height: 22px;
         }
 
-        a {
+        a,
+        i {
           display: inline-block;
           width: 100%;
           color: @text-color-secondary;
```
Comment (afc163) [inline components/card/style/index.less]: 这边需要补上 `transition: color 0.3s;`

Comment (lhx6538665) [reply components/card/style/index.less]: 已经加上了

**Comment by afc163** [comment]
ci 挂了，rebase 一下 master。

**Comment by lhx6538665** [comment]
是不是被我弄坏了，我没多少经验，要不我重新开一个pr？

**Comment by afc163** [comment]
..重开吧

**Pull request closed without merging**